enf/endpoint_ratelimit.go

```diff
@@ -0,0 +1,67 @@
+package enf
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+)
+
+// EndpointService handles communication with all endpoint related methods within
+// the ENF API.
+type EndpointService service
+
+// EndpointRateLimits represents the values of rate limits for an endpoint.
+type EndpointRateLimits struct {
+	PacketsPerSecond *int  `json:"packets_per_second"`
+	PacketsBurstSize *int  `json:"packets_burst_size"`
+	BytesPerSecond   *int  `json:"bytes_per_second"`
+	BytesBurstSize   *int  `json:"bytes_burst_size"`
+	Inherit          *bool `json:"inherit"`
+}
+
+// endpointRateLimitResponse represents the standard API response for
+// all relevant endpoint rate limit endpoints.
+type endpointRateLimitResponse struct {
+	Data []*EndpointRateLimits  `json:"data"`
+	Page map[string]interface{} `json:"page"`
+}
+
+// GetCurrentRateLimits gets the current rate limits for the given endpoint IPv6 address.
+func (s *EndpointService) GetCurrentRateLimits(ctx context.Context, endpointIPv6 string) (*EndpointRateLimits, *http.Response, error) {
```

Comment:
Should we drop current from the function names?  Just `GetRateLimit` and `GetMaxRateLimit`?